src/NUnitFramework/tests/Internal/Filters/NotFilterTests.cs

```diff
@@ -54,7 +54,7 @@ public void PassTest()
         {
             var filter = new NotFilter(new CategoryFilter("Dummy"));
 
-            Assert.False(filter.Pass(_topLevelSuite));
+            Assert.True(filter.Pass(_topLevelSuite));
             Assert.False(filter.Pass(_dummyFixture));
             Assert.False(filter.Pass(_dummyFixture.Tests[0]));
```

Comment:
Yes, exactly. Knowing the implementation, I realize that the test is likely to be redundant. But the implementation can change at any time and if there's no test like that then nothing will fail.

There were two different error reports, one of them having two test methods in one fixture.